src/lightgbm/src/test/resources/benchmarks_VerifyLightGBMRegressor.csv

```diff
@@ -0,0 +1,6 @@
+name,value,precision,higherIsBetter
+LightGBMRegressor_/home/marhamil/lib/datasets/Regression/Train/energyefficiency2012_data.train.csv,4.04001976483622,1.0,true
```

Comment:
I see the full path (including marhamil username here), looks like this needs to be fixed

Reply:
thanks!